pkg/plugins/platform/local.go

```diff
@@ -232,7 +232,7 @@ func (l *LocalApp) ImportDB(imPath string, extPath string) error {
 		return fmt.Errorf("no .sql files found to import")
 	}
 
-	fmt.Println("Importing database...")
+	fmt.Printf("Importing database from %s...\n", importPath)
```

Comment:
This is seen by users, and outputs the absolute path, resulting in a long line of output. Perhaps a debug output instead?